CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Fixed
```

Comment:
How about _Enhancements and fixes_?

Reply:
Enhancements would be in "Added".

We follow https://keepachangelog.com/en/1.0.0/